Report/sections/4_recapitulation/discussion.tex

```diff
@@ -0,0 +1,45 @@
+\chapter{Discussion}
+Our goal with this project was to design and implement a compiler for a programming language for beginners in the context of game development. 
+This sections seeks to communicate the circumstances under which the project was developed as well as give insight into our thoughts about the end result.
+We also wish to elaborate on specific decisions that we, in retrospect, feel were poor or overly ambitious.
+
+\section{Ambitiousness}
+A major challenge in the first half of the project that ended up impacting the project as a whole was that our original group had a difference in ambition. 
+One half of the group had lower expectations which caused conflicts and limited the group work.
+Much time was spent attemping to solve these conflicts and getting everyone to agree on how much effort and time to invest in the project.
+This combined with the overall level of ambition meant that we fell behind as not much work was being done on the project for the first two months. 
+Eventually, this resulted in the decision to split our group in two halves as we were unable to find an agreeable compromise.
+
+Because we stuck to the original vision for the project despite having less time and a smaller group, we unfortunately ended up investing too much time into features that we realistically did not have time to implement such as member access.
```

Comment:
```suggestion
Because we stuck to the original vision for the project despite having less time and a smaller group, we unfortunately ended up investing too much time into features that we realistically did not have time to implement - such as member access.
```

Reply:
Or at least some other way of mentioning it by separating it from the sentence

Reply:
Maybe even remove it. It's kind of weird to have there - especially when you get to line 22. Feels out of place.

```suggestion
Because we stuck to the original vision for the project despite having less time and a smaller group, we unfortunately ended up investing too much time into features that we realistically did not have time to implement.
```